receiver/githubactionsreceiver/trace_attributes.go

```diff
@@ -52,10 +52,8 @@ func createResourceAttributes(resource pcommon.Resource, event interface{}, conf
 		attrs.PutStr("ci.github.workflow.job.started_at", e.GetWorkflowJob().GetStartedAt().Format(time.RFC3339))
 		attrs.PutStr("ci.github.workflow.job.status", e.GetWorkflowJob().GetStatus())
 
-		// Grafana self-hosted runners have a Group Name of Default (for prod) and Self-hosted (for dev)
-		// TODO: Update this to only look for self-hosted once we've deployed the change to our prod runners
 		rg := strings.ToLower(e.GetWorkflowJob().GetRunnerGroupName())
-		if rg == "default" || rg == "self-hosted" {
```

Comment:
"default" is no longer relevant here?

Reply:
not sure if we need it anymore - it's just one runner as per the image, and I cannot see it being used 👀 
<img width="923" height="104" alt="Screenshot 2025-08-25 at 3 30 29 PM" src="https://github.com/user-attachments/assets/d32fe83a-2d62-4159-81e3-31f7e0974b61" />

Reply:
okidoki :)